src/data/queries/formattedCitationQuery.ts

```diff
@@ -23,12 +37,39 @@ export const FORMATTED_CITATION_GQL = gql`
   }
 `
 
-export interface QueryVar {
-  id: string
-  style: string
-  locale?: string
+// Direct API fetch implementation
+async function fetchFormattedCitation(variables: QueryVar): Promise<string> {
+  const params = new URLSearchParams({
+    style: variables.style,
+    lang: variables.locale || 'en-US'
+  })
+
+  const url = `${BASE_CITATION_URL}${variables.id}?${params.toString()}`
+
+  const response = await fetch(url)
+
+  if (!response.ok) {
+    throw new Error(`Citation API returned ${response.status}: ${response.statusText}`)
+  }
+
+  const text = await response.text()
+  return text
 }
 
-export interface QueryData {
-  work: FormattedCitation
+// React Query hook
+export function useDirectCitationQuery(variables: QueryVar): UseQueryResult<QueryData, Error> {
+  return useReactQuery({
+    queryKey: ['citation', variables.id, variables.style, variables.locale],
```

Comment:
tanstack query can accept objects in the query key array, you may want to simply use `variables` unless you had a specific reason for breaking it up like this

Reply:
Will do